src/core/common/logos/types.ts

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ColorScheme, ImageType } from './constants';
+
+/**
+ * @public
+ */
+export interface Logos {
+  /**
+   * @deprecated Use {@link Logos.Application} instead.
```

Comment:
Are we missing the `Application` prop in this interface?

Reply:
Yeah; i hadn't pushed the complete change. After our convo last week, `Logos.OpenSearchDashboards` made no sense and we decided to have it renamed to `Logos.Application`. I will push the changes momentarily.